pkg/cli/token/token.go

```diff
@@ -149,6 +151,13 @@ func Rotate(app *cli.Context) error {
 	if err := cmds.InitLogging(); err != nil {
 		return err
 	}
+	if !app.Bool("force") {
+		warning := fmt.Sprint("\033[33mWARNING\033[0m: This will replace the existing token with a new one.",
```

Comment:
I'm a little iffy on control codes and input prompts in our CLI. Does kubectl have a pattern for this that we can reuse?

Reply:
This was a request from @Oats87. There isn't really a existing usage in kubectl that ever asks for user confirmation. I see several alternatives:
1) Just print the warning, the other server nodes still have the old token, you just entered the old token to execute this command, we are just warning the user to remember/save it somewhere. 
2) Print the warning, with a countdown of 10sec so the operator could manually cancel the action before the old token is removed. An automated system will simply wait the 10 sec.